graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -510,7 +519,7 @@ def test_graph_nodeProperties_stream_with_arrow_separate_property_columns(gds: G
     result = gds.graph.nodeProperties.stream(
         G, ["x", "y"], db_node_properties=["z", "name"], separate_property_columns=True, concurrency=2
     )
-    assert list(result.keys()) == ["nodeId", "x", "y", "z", "name"]
+    assert set(result.keys()) == {"nodeId", "x", "y", "z", "name"}
```

Comment:
why did you change this assertion? if `result.keys()` contains a duplicate, then it will still pass this test. is that intentional?

Reply:
i think we shouldnt care about the ordering of the columns here.